cmd/konvoy-image-wrapper/cmd/wrapper.go

```diff
@@ -44,6 +44,9 @@ const (
 	envVSpherePassword                   = "VSPHERE_PASSWORD"
 	envRedHatSubscriptionManagerUser     = "RHSM_USER"
 	envRedHatSubscriptionManagerPassword = "RHSM_PASS"
+	envVSphereSSHUserName                = "SSH_USERNAME"
+	envVSphereSSHPassword                = "SSH_PASSWORD"
+	envVsphereSSHPrivatekeyFile          = "SSH_PRIVATE_KEY_FILE"
```

Comment:
This also needs to be mounted into the container, similar to https://github.com/mesosphere/konvoy-image-builder/blob/551eb74f30685f357836ef06615cb65efe7553a6/cmd/konvoy-image-wrapper/cmd/wrapper.go#L241

Reply:
we have `VSPHERE_USERNAME` and `VSPHERE_PASSWORD` to authenticate with vsphere API. I thought adding `VSPHERE` prefix might add confusion. I am happy to change to name it differently if you have any suggestion. 
Also, packer creates temporary keypair for other providers per run, AFIK, other providers would not need these parameters.

Reply:
> This also needs to be mounted into the container, similar to

Thank you for catching this.

Reply:
I have to refactor a code a bit to remove repetition in the code. https://github.com/mesosphere/konvoy-image-builder/pull/552/commits/0f1900c6797bb6d8b7b38f014f8f4a82a8615e06 I have added unit tests to make sure mounting works. Filing another stacked PR to change CI jobs to use wrapper so that we can make sure that it does not break existing functionality. blocking this PR meanwhile.